packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPVideoPlayerPlugin.m

```diff
@@ -397,7 +399,15 @@ - (void)updatePlayingState {
     return;
   }
   if (_isPlaying) {
-    [_player play];
+    // Calling play is the same as setting the rate to 1.0 (or to defaultRate depending on ios
+    // version) so last set playback speed must be set here if any instead.
+    // https://github.com/flutter/flutter/issues/71264
+    // https://github.com/flutter/flutter/issues/73643
+    if (_playbackSpeed) {
+      [self updateRate];
+    } else {
+      [_player play];
```

Comment:
Wouldn't the codepaths be simpler at this point if we just always used `rate` instead of `play`/`pause`?

Reply:
That is to potentially support `defaultRate` when `_playbackSpeed` is unset, there can be new function like `setDefaultPlaybackSpeed` and support from dart side, or maybe speed set by `setPlaybackSpeed` can be relative to `defaultRate` but that would require checking version of ios.

Reply:
Oh, I see. Sorry, I had forgotten about the part of the PR description explaining that the official docs for `play`/`pause` are actually wrong now.